test/support/model_case.ex

```diff
@@ -59,7 +59,7 @@ defmodule OpenPantry.ModelCase do
   """
   def errors_on(struct, data) do
     struct.__struct__.changeset(struct, data)
-    |> Ecto.Changeset.traverse_errors(&OpenPantry.ErrorHelpers.translate_error/1)
+    |> Ecto.Changeset.traverse_errors(&OpenPantry.Web.ErrorHelpers.translate_error/1)
```

Comment:
Oh, was this just broken/wrong? I don't see corresponding namespace change in ErrorHelpers so I assume so but would think it wouldn't compile then?